modules/caas/api/src/main/java/io/cattle/platform/api/service/ServiceCreateValidationFilter.java

```diff
@@ -367,14 +368,30 @@ protected void validateIpsHostName(ApiRequest request) {
         }
     }
 
-    protected void validateLaunchConfigs(Service service, ApiRequest request) {
+    @SuppressWarnings("unchecked")
+    protected ApiRequest validateLaunchConfigs(Service service, ApiRequest request) {
         Map<String, Object> data = CollectionUtils.toMap(request.getRequestObject());
+
+        List<Object> modifiedSlcs = new ArrayList<>();
+        if (data.get(ServiceConstants.FIELD_SECONDARY_LAUNCH_CONFIGS) != null) {
```

Comment:
From a code readability perspective it would be nice of this logic was in a method like `validationNetworkMode`